### ISSUE TITLE
Issue on default scrap execution

### ISSUE THREAD

**Issue by JuanCarlosCamara** [David-Carrasco/Scrapy-Idealista#11]
Hello, 

I´m trying to execute default scrapping from Carabanchel and I´m receiving errors about dead proxies. 

I have tried to upload the DOWNLOAD_TIMEOUT parameter in settings.py from 10 to 20 seconds as I have read in other issues and it still returns same error.

Do you have any idea or what could be happening? I don´t know if maybe idealista has included some kind of check or security to avoid scrapping since October.

Thanks a lot for your help.
Best regards,
Juan Carlos Cámara

**Comment by VictorUceda** [comment]
same error today

**Comment by abidhsn** [comment]
Facing the exact same issue...the proxies are dead

**Comment by somefreestring** [comment]
yes all the proxies are dead, removing the middleware seems to ban you in minutes

**Comment by ewoks** [comment]
@David-Carrasco is this expected or temporal issue? Do we do something wrong? Thanks

**Comment by foodaka** [comment]
Any idea on how to fix this?

**Comment by David-Carrasco** [comment]
I've tested the scraper again as @JuanCarlosCamara said, it seems Idealista has added some check to block requests of all the proxies which come from https://free-proxy-list.net/

The idea is to provide another list of proxies to be used by the scraper declared in:

https://github.com/David-Carrasco/Scrapy-Idealista/blob/afc34add09d6f9b191bcc23a033c89ab4b1b46b8/idealista/settings.py#L62

It should work since I've tested the scraper with my own ip and it works. It's a proxies issue.

**Comment by agusriscos** [comment]
I am still stuck with the same problem. Has anyone got a valid proxy list?

**Comment by IsmaPons** [comment]
hello, what about this?

https://github.com/clarketm/proxy-list/blob/master/proxy-list.txt

We'd still need to modify the code though

**Comment by mikemajara** [comment]
@IsmaPons doesn't seem to be working either. I'm curious to know what the result was back in your first attempt. I'm getting DEAD proxies all over the place. Just now testing [this one out](https://github.com/stamparm/fetch-some-proxies).

**Comment by mikemajara** [comment]
Apparently the problem is not the proxies themselves (several other lists which are healthy don't work either). The issue here seems to be a 403 returned as the the bot is detected 🤖 . So the proxy middleware keeps trying with no success but it's really the HTTP 403 whats blocking you from success. Even when all settings and variables are correctly set, Google Analytics has apparently has an anti-bot feature 🧐.

Long story short, try
- changing the USER-AGENT
- installing selenium or using with scrapy-splash

Useful links: [SO answer](https://stackoverflow.com/a/47319968/3157057), [Splash](https://www.zyte.com/blog/handling-javascript-in-scrapy-with-splash/), [User-Agent](https://coderslegacy.com/python/scrapy-user-agents/).

**Comment by davidrdguezarias** [comment]
@mikemajara @David-Carrasco did you test with the proposed solution on 7th November? Is it working?

**Comment by mikemajara** [comment]
I did to no avail. It did work once. But it's hard to get around it.

**Comment by BeMoAcademic** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](http://bit.ly/3SqCGvw)

🚀 **Exciting Announcement!** 

AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 400,000,000 (4% of the total supply)

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](http://bit.ly/3SqCGvw).

The window to [Claim $ALT tokens](http://bit.ly/3SqCGvw) is open from January 25 to February 08, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

Chenghao-Wu, @psemdel, @nabiuddin6, @j20963j2000, @mingbackmountain, @seanmckdev, @sumitbehal, @tchan09, @eth-man, @jpforny, @CryptoKR, @JaylonNichols, @miali88, @TristanoB, @nitinongit, @Frenda, @martinandrovich, @dmccue, @FF99CC, @NoelJacob
